Flow.Launcher/ResultListBox.xaml

```diff
@@ -30,55 +37,102 @@
+                        <StackPanel
+                            Grid.Column="2"
+                            Margin="0,0,10,0"
+                            Visibility="{Binding ShowOpenResultHotkey}">
+                            <TextBlock
+                                x:Name="Hotkey"
+                                Margin="12,0,12,0"
+                                Padding="0,10,0,10"
+                                HorizontalAlignment="Right"
+                                VerticalAlignment="Center"
+                                Opacity="0.8"
+                                Style="{DynamicResource ItemHotkeyStyle}">
                                 <TextBlock.Visibility>
-                                    <Binding RelativeSource="{RelativeSource Mode=FindAncestor, AncestorType=ListBoxItem}" Converter="{StaticResource ResourceKey=OpenResultHotkeyVisibilityConverter}" />
+                                    <Binding Converter="{StaticResource ResourceKey=OpenResultHotkeyVisibilityConverter}" RelativeSource="{RelativeSource Mode=FindAncestor, AncestorType=ListBoxItem}" />
                                 </TextBlock.Visibility>
                                 <TextBlock.Text>
                                     <MultiBinding StringFormat="{}{0}+{1}">
                                         <Binding Path="OpenResultModifiers" />
-                                        <Binding RelativeSource="{RelativeSource Mode=FindAncestor, AncestorType=ListBoxItem}" Converter="{StaticResource ResourceKey=OrdinalConverter}" />
+                                        <Binding Converter="{StaticResource ResourceKey=OrdinalConverter}" RelativeSource="{RelativeSource Mode=FindAncestor, AncestorType=ListBoxItem}" />
                                     </MultiBinding>
                                 </TextBlock.Text>
                             </TextBlock>
                         </StackPanel>
-                        <Border BorderThickness="0" BorderBrush="Transparent" Margin="9 0 0 0">
-                            <Image x:Name="ImageIcon" Width="32" Height="32" HorizontalAlignment="Center" Source="{Binding Image}" Visibility="{Binding ShowIcon}"  Margin="0 0 0 0" Stretch="UniformToFill"/>
+                        <Border
+                            Margin="9,0,0,0"
+                            BorderBrush="Transparent"
+                            BorderThickness="0">
+                            <Image
+                                x:Name="ImageIcon"
+                                Width="32"
+                                Height="32"
+                                Margin="0,0,0,0"
+                                HorizontalAlignment="Center"
+                                Source="{Binding Image}"
+                                Stretch="Uniform"
```

Comment:
It's not related to this pr, but I added it as a minor correction. Corrects the problem of being cut when the icon size is large (cloud icon of the e.g plug-in manager). When I fixed this part before, the icon was small, so there seemed to be no problem.